content/en/docs/setup/install/operator/index.md

```diff
@@ -75,11 +73,197 @@ accessible to `istioctl` by using this command:
 $ istioctl experimental profile list
 {{< /text >}}
 
-Optionally, you can use the `-s` flag with a install package path to see the
-list of configuration profiles available for other Istio versions:
+## Customize Istio settings using the IstioControlPlane API
+
+You can change a feature or component setting by using the [IstioControlPlane API](https://github.com/istio/operator/blob/release-1.3/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto).
+
+### Identify the feature or component settings
+
+The API groups Istio control plane components by feature, as shown in the table below: 
+
+| Feature | Components |
+|---------|------------|
+Base | CRDs
+Traffic Management | Pilot
+Policy | Policy
+Telemetry | Telemetry
+Security | Citadel
+Security | Node agent
+Security | Cert manager
+Configuration management | Galley
+Gateways | Ingress gateway
+Gateways | Egress gateway
+AutoInjection | Sidecar injector
+
+In addition to the core Istio components, third-party addon features and components are also available:
+
+| Feature | Components |
+|---------|------------|
+Telemetry | Prometheus
+Telemetry | Prometheus Operator
+Telemetry | Grafana
+Telemetry | Kiali
+Telemetry | Tracing
+ThirdParty | CNI
+
+Features can be enabled or disabled, which enables or disables all of the components that are a part of the feature.
```

Comment:
just a note for future, CNI is core, albiet alpha or beta, not ThirdParty.  I'd just call it CNI in the future.

Reply:
Created https://github.com/istio/istio/issues/16955.